### DISCUSSION TITLE
Unable to install or uninstall desktop v18.0.1 in Windows 11

### DISCUSSION THREAD

**Discussion by JHCairn** [jgraph/drawio-desktop#990] [General]
The install for desktop is not completing for 18.0.1 on windows 11. Nor can I uninstall it. Getting a message about needing to be an admin. I AM an admin on my laptop. Using web for now. Thanks!

**Comment by davidjgraph** [comment]
Did you run the installer as admin specifically?

**Reply by JHCairn** [reply]
i did not. I will try that.

**Reply by JHCairn** [reply]
No joy. I right clicked on the installer package in my downloads and said run as Admin. Then i confirmed the install location. I actually changed it to a new one in case something with the earlier version in the default location was causing an issue. Then I confirmed OK and continued. Then i got the message in the attached screenshot. I double checked draw.io was not already running or open anywhere in Task Manager (which it wasn't). Hitting retry a few times resulted in the same looping message, so I hit cancel instead. 
![image](https://user-images.githubusercontent.com/33914698/167110555-3cd428b2-2997-46e6-b5eb-e04e7e9fb06a.png)

**Reply by JHCairn** [reply]
tried again with the default install location (running as admin) and got the same experience although it ran longer in time before showing the error dialog above.

**Reply by davidjgraph** [reply]
Exactly which binary did you use?

**Reply by JHCairn** [reply]
I was using the latest version available before logging off on Friday. I grabbed new file this morning and tried again with SUCCESS. THANK YOU!
